integration_test/README.md

```diff
@@ -42,18 +42,15 @@ of Kokoro with some setup (see above).
 ### Testing Command
 
 When the setup steps are complete, you can run ops_agent_test (for Linux)
-like this:
-
+from the Makefile:
```

Comment:
Thought: Can we add a link to the task.mak here? So that we can easily navigate to and back from the makefile while reading the doc.